cmd/build/command.go

```diff
@@ -80,21 +83,29 @@ func Build(ctx context.Context) *cobra.Command {
 	return cmd
 }
 
-func (bc *Command) getManifestAndBuildVersion(options *types.BuildOptions) (*model.Manifest, bool) {
-	manifest, errManifest := bc.GetManifest(options.File)
-	if errManifest != nil {
-		oktetoLog.Debug("error getting manifest v2 from file %s: %v. Fallback to build v1", options.File, errManifest)
+func (bc *Command) getBuilder(options *types.BuildOptions) (Builder, error) {
+	var builder Builder
+
+	manifest, err := bc.GetManifest(options.File)
+	if err != nil {
+		if errors.Is(err, oktetoErrors.ErrInvalidManifest) {
```

Comment:
I have changed the error handling logic a bit. Since an invalid manifest error can not be comparable with == because the cause of the error is unknown a priori.